### ISSUE TITLE
Tag a release on GitHub?

### ISSUE THREAD

**Issue by pramodk** [neuronsimulator/iv#30]
Traditionally we have releases under https://neuron.yale.edu/ftp/neuron/versions/v7.6/. May be we can also start tagging releases on GitHub so that we can use those for packaging softwares?

**Comment by nrnhines** [comment]
I agree that needs to be done. Also some association needs to be developed that indicates compatible
versions between nrn and iv (perhaps similar to nrn and coreneuron).
